src/js/index.js

```diff
@@ -36,18 +36,23 @@ const printAllTasks = () => {
       taskmanager.editTask(task);
     });
```

Comment:
- [ ] Please as demanded by the project requirements, the functionalities for the status updates (completed: true / false) should be in a separate js file. Consider moving this to a new method in a new js file and import then call the method here.
PS: **I couldn't comment on the particular lines that's why I commented below but I am talking about line 31-37**